forge-gui/res/cardsfolder/upcoming/slimefoot_and_squee.txt

```diff
@@ -0,0 +1,13 @@
+Name:Slimefoot and Squee
+ManaCost:B R G
+Types:Legendary Creature Fungus Goblin
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, create a 1/1 green Saproling creature token.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigToken | Secondary$ True | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, create a 1/1 green Saproling creature token.
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_saproling
+A:AB$ ChangeZone | Cost$ 1 B R G Sac<1/Saproling> | Origin$ Graveyard | Destination$ Battlefield | ActivationZone$ Graveyard | ValidTgts$ Creature.YouCtrl+Other | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target other creature card in your graveyard | SorcerySpeed$ True | SubAbility$ DBChangeZone | SpellDescription$ Return CARDNAME and up to one other target creature card from your graveyard to the battlefield. Activate only as a sorcery.
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Defined$ Self | Destination$ Battlefield

```

Comment:
Is there a way to do this with just one ChangeZone line? If not, should we update the engine to allow it? @tool4ever 

Seems wrong that you move target first and then Self (opposite of oracle)

Reply:
I might just update ChangeZone for this and Snow Hound, Coastal Wizard, etc.?

Reply:
yea please do 🧙 
it matters for triggers!

Reply:
@Northmoc Thanks for the changes. Will change this one according to your PR